src/styles/blocks/menu.scss

```diff
@@ -0,0 +1,50 @@
+.menu {
+  box-sizing: border-box;
```

Comment:
we can remove it because we already define `box-sizing: border-box` in normalize